docs/docs/guides/dashboard/on-premise/install.md

```diff
@@ -131,6 +132,20 @@ The environment variables required to authenticate against S3-compliant storages
 ### Google Cloud Storage
 For Google Cloud Storage, follow [these docs](https://cloud.google.com/storage/docs/authentication/managing-hmackeys) to get the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` pair. The `AWS_ENDPOINT` should be set to `https://storage.googleapis.com`. Other environment variables are the same as for any other S3-compliant storage.
 
+### Version control system configuration
+
+We integrate with providers of your version control system to provide extra features such as posting automatically posting comments in your pull requests.
+
+#### GitHub
+
+You will need to create a GitHub app. You can reuse the one you created for authentication. In the `Permissions and events`'s `Repository permissions` section, you will need to additionally set the `Pull requests` permission to `Read and write`.
+
+On top of the `TUIST_GITHUB_APP_CLIENT_ID` and `TUIST_GITHUB_APP_CLIENT_SECRET`, you will need the following environment variables:
+
+| Environment variable | Description | Required | Default | Example |
+| --- | --- | --- | --- | --- |
+| `TUIST_GITHUB_APP_PRIVATE_KEY` | The private key of the GitHub application | Yes | | `-----BEGIN RSA PRIVATE KEY-----...` |
+
```

Comment:
I'd add that if they plan to use this integration, they create a `GitHub App` instead of a `GitHub OAuth App`. The former can be used for authorization purposes too.